api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/TaskTranslatorImplTest.java

```diff
@@ -946,6 +958,174 @@ public void toOpenmrsType_shouldTranslateMultipleInputValueTypes() {
 		
 	}
 	
+	// Task Code
+	@Test
+	public void toFhirResource_shouldTranslateTaskCode() {
+		
+		FhirTask task = new FhirTask();
+		Concept openmrsObject = new Concept();
+		CodeableConcept fhirObject = newCodeableConcept();
+		when(conceptTranslator.toFhirResource(openmrsObject)).thenReturn(fhirObject);
+		
+		task.setTaskCode(openmrsObject);
+		Task result = taskTranslator.toFhirResource(task);
+		assertThat(result.getCode(), CoreMatchers.notNullValue());
+		assertThat(result.getCode(), CoreMatchers.equalTo(fhirObject));
```

Comment:
```suggestion
		FhirTask task = new FhirTask();
		Concept openmrsObject = new Concept();
		CodeableConcept fhirObject = newCodeableConcept();
		when(conceptTranslator.toFhirResource(openmrsObject)).thenReturn(fhirObject);
		task.setTaskCode(openmrsObject);
		
		Task result = taskTranslator.toFhirResource(task);
		
		assertThat(result.getCode(), CoreMatchers.notNullValue());
		assertThat(result.getCode(), CoreMatchers.equalTo(fhirObject));
```

General rule here is to break things (visually) into: setup / execution / assertions. It makes the tests easier to tell what they do at a glance. This formatting should be adopted for all the tests in this file.